pulser-core/pulser/register/_reg_drawer.py

```diff
@@ -17,7 +17,7 @@
 from collections import defaultdict
 from collections.abc import Sequence as abcSequence
 from itertools import combinations
-from typing import Optional
+from typing import Mapping, Optional
```

Comment:
I think you're supposed to import `Mapping` from `collections.abc` (importing from `typing` has been deprecated).